apps/condo/migrations/20230427172150-0264_newsitem_ispublished_newsitem_sentat_and_more.js

```diff
@@ -0,0 +1,51 @@
+// auto generated by kmigrator
+// KMIGRATOR:0264_newsitem_ispublished_newsitem_sentat_and_more:IyBHZW5lcmF0ZWQgYnkgRGphbmdvIDQuMC4xIG9uIDIwMjMtMDQtMjcgMTI6MjIKCmZyb20gZGphbmdvLmRiIGltcG9ydCBtaWdyYXRpb25zLCBtb2RlbHMKCgpjbGFzcyBNaWdyYXRpb24obWlncmF0aW9ucy5NaWdyYXRpb24pOgoKICAgIGRlcGVuZGVuY2llcyA9IFsKICAgICAgICAoJ19kamFuZ29fc2NoZW1hJywgJzAyNjNfaW5jaWRlbnRleHBvcnR0YXNraGlzdG9yeXJlY29yZF9pbmNpZGVudGV4cG9ydHRhc2snKSwKICAgIF0KCiAgICBvcGVyYXRpb25zID0gWwogICAgICAgIG1pZ3JhdGlvbnMuQWRkRmllbGQoCiAgICAgICAgICAgIG1vZGVsX25hbWU9J25ld3NpdGVtJywKICAgICAgICAgICAgbmFtZT0naXNQdWJsaXNoZWQnLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuQm9vbGVhbkZpZWxkKGRlZmF1bHQ9RmFsc2UpLAogICAgICAgICAgICBwcmVzZXJ2ZV9kZWZhdWx0PUZhbHNlLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5BZGRGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nbmV3c2l0ZW0nLAogICAgICAgICAgICBuYW1lPSdzZW50QXQnLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuRGF0ZVRpbWVGaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICAgICAgbWlncmF0aW9ucy5BZGRGaWVsZCgKICAgICAgICAgICAgbW9kZWxfbmFtZT0nbmV3c2l0ZW1oaXN0b3J5cmVjb3JkJywKICAgICAgICAgICAgbmFtZT0naXNQdWJsaXNoZWQnLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuQm9vbGVhbkZpZWxkKGJsYW5rPVRydWUsIG51bGw9VHJ1ZSksCiAgICAgICAgKSwKICAgICAgICBtaWdyYXRpb25zLkFkZEZpZWxkKAogICAgICAgICAgICBtb2RlbF9uYW1lPSduZXdzaXRlbWhpc3RvcnlyZWNvcmQnLAogICAgICAgICAgICBuYW1lPSdzZW50QXQnLAogICAgICAgICAgICBmaWVsZD1tb2RlbHMuRGF0ZVRpbWVGaWVsZChibGFuaz1UcnVlLCBudWxsPVRydWUpLAogICAgICAgICksCiAgICBdCg==
+
+exports.up = async (knex) => {
+    await knex.raw(`
+    BEGIN;
+--
+-- Add field isPublished to newsitem
+--
+ALTER TABLE "NewsItem" ADD COLUMN "isPublished" boolean DEFAULT false NOT NULL;
+ALTER TABLE "NewsItem" ALTER COLUMN "isPublished" DROP DEFAULT;
+--
+-- Add field sentAt to newsitem
+--
+ALTER TABLE "NewsItem" ADD COLUMN "sentAt" timestamp with time zone NULL;
```

Comment:
please add indexes

Reply:
This is an alteration. Indexes already created.
<img width="463" alt="image" src="https://user-images.githubusercontent.com/25384290/234876607-956daf6a-88bf-4f49-98de-fef0cb3e5562.png">